POC/ColorPicker/ColorPicker/MainWindow.xaml.cs

```diff
@@ -80,6 +93,31 @@ private void OnNewColorButtonClick(object sender, EventArgs e)
             }
         }
 
+        private void OnTextBoxPreviewMouseLeftButtonDown(object sender, MouseButtonEventArgs e)
+        {
+            // Source: https://www.intertech.com/Blog/how-to-select-all-text-in-a-wpf-textbox-on-focus/
+            // and https://stackoverflow.com/questions/660554/how-to-automatically-select-all-text-on-focus-in-wpf-textbox
+            TextBox textBox = sender as TextBox;
+            if (textBox != null && !textBox.IsKeyboardFocusWithin)
+            {
+                if (e.OriginalSource.GetType().Name == "TextBoxView")
+                {
+                    e.Handled = true;
+                    textBox.Focus();
+                }
+            }
+        }
+
+        private void OnTextBoxFocus(object sender, RoutedEventArgs e)
+        {
+            TextBox textBox = e.OriginalSource as TextBox;
+            if (textBox != null)
+            {
+                textBox.SelectAll();
+                CopyToClipboard(textBox);
```

Comment:
Is this supposed to be 
```suggestion
                CopyToClipboardAndOpenPopup(textBox);
```

Reply:
I'm sorry, I must have missed that. I thought it compiled 🙃